core/src/main/scripts/importer/validateData.py

```diff
@@ -4765,7 +4765,7 @@ def validateStudyTags(tags_file_path, logger):
             extra={'filename_': tags_file_path})
         with open(tags_file_path, 'r') as stream:
             try:
-                parsedYaml = yaml.load(stream)
+                parsedYaml = yaml.load(stream, Loader=yaml.FullLoader)
```

Comment:
I think this correction does not work in CI. CI runs still on python 3.4. I propose to remove this for now.

Reply:
hmm in my other PR I also adjusted this in the travis file to 3.6

Reply:
But here you did? Or did I miss something.

Reply:
But here you did? Or did I miss something.

Reply:
please see next commit :)